tests/vmi_configuration_test.go

```diff
@@ -2378,6 +2378,16 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 					DedicatedCPUPlacement: true,
 					IsolateEmulatorThread: true,
 				}
+				cpuVmi.Spec.Domain.Resources = v1.ResourceRequirements{
```

Comment:
Maybe worth a separate test as there are 2 ways how to ask for this and only one of those where broken.